common/alert/alerts-en.html

```diff
@@ -0,0 +1,154 @@
+---
+{
+	"title": "GCWeb Alert example",
+	"language": "en",
+	"altLangPage": "alerts-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-02-28"
+}
+---
+
+<h2>Alert with headings</h2>
+<section class="alert alert-danger">
+	<h3>Danger alert</h3>
+	<p>Optional alert details.</p>
```

Comment:
This change to be applied to the other alert variant.

```suggestion
	<p>Alert details.</p>
```

Semantically we can not have a section that consist into being a single heading. An heading is always followed with content or inner sections.